### PR TITLE
Proposing adding some other error codes

### PR DESCRIPTION
**Pull request by get-itips**
@JoanneHendrickson please review if you agree

https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/1491

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.
--

File: migration/troubleshooting-common-spmt-issues.md [hunk 1]
```diff
@@ -82,7 +82,8 @@ This article describes some common issues and errors that you may encounter when
 |0x02070009|Several packages failed to upload. Pause the task and check your network connection.|
 |0x01710009|A failure occured due to job end failures; some items failed in the package. Restart migration.|
 0x01610001|The Azure container is expired. Retry migration task.|   
-
+|0x01710004|Errors or timeout for Server Processing the file. Fail to lookup folder name. The item may exist in other list or site in the same site collection. Or the item is in the recycle bin.|
```
Comment (JohanFreelancer9) [inline migration/troubleshooting-common-spmt-issues.md]:
```suggestion
|0x01710004|Errors or timeout of the server processing the file. Failed to look up folder name. The item may exist in another list or site in the same site collection, or the item is in the recycle bin.|
```

Comment (get-itips) [reply migration/troubleshooting-common-spmt-issues.md]: Hello Johan, the sentences used here were contributed by a user who experienced those errors, so he received the exact same words from the application, my opinion would be to maintain the exact sentences...

Comment (JohanFreelancer9) [reply migration/troubleshooting-common-spmt-issues.md]: @get-itips Fair enough. I will delete my suggestion. Thank you for clarifying.

Comment (JohanFreelancer9) [inline migration/troubleshooting-common-spmt-issues.md]:
```suggestion
|0x01710004|Errors or timeout for Server Processing the file. Fail to lookup folder name. The item may exist in other list or site in the same site collection. Or the item is in the recycle bin.|
```

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@JoanneHendrickson can you review my pull request? Thanks.

**Pull request merged**